django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -19,6 +19,7 @@
 
 .selector-chosen select {
     border-top: none;
+    height: 20.8em;
```

Comment:
The mismatched height issue is present in both LTR and RTL versions so we might want to treat it as a separate issue (or at least as two commits, as Tim said).

Having a comment explain how you came up with that oddly specific value would be helpful to future maintainers.